backend/pkg/helper/validation.go

```diff
@@ -0,0 +1,10 @@
+package helper
+
+func EnsureNotEmpty(fields ...string) error {
+	for _, field := range fields {
+		if field == "" {
+			return NewMicroMissingParameterErr(NotificationsServiceID)
+		}
+	}
+	return nil
+}
```

Comment:
To allow this function to be more re-usable, you can change it to return a boolean and then construct an error where you need it

```suggestion
func SomeEmpty(fields ...string) bool {
	for _, field := range fields {
		if field == "" {
			return true
		}
	}
	return false
}
```

```go
// ...
if helper.SomeEmpty(myStrings...) {
	return NewMicroMissingParameterErr(NotificationsServiceID)
}
// ...
```

Reply:
Implemented in 2a3ba4b